quickstarts/azure/azure-iot-central-applications/config.yml

```diff
@@ -11,7 +11,7 @@ description: |-
 
   ### Why monitor Azure IoT Central Applications with New Relic?
 
-  [New Relic Azure IoT Central Applications](https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-iot-central-applications-monitoring-integration/) monitoring quickstart empowers you to track the performance of Azure IoT Central Applications via different metrics including c2d commands success, connected device count, data export error and device data usage. 
+  [New Relic Azure IoT Central Applications](https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-iot-central-application-monitoring-integration/) monitoring quickstart empowers you to track the performance of Azure IoT Central Applications via different metrics including c2d commands success, connected device count, data export error and device data usage. 
```

Comment:
This link doesn't seem to work/be live - is there an expectation for this doc to go live at a certain time/date?

Reply:
I just found another problem - there is a typo in the filename for docs. Filing another PR. It will be fixed shortly, but not until GitHub recovers (currently experiencing outages).

Reply:
https://github.com/newrelic/docs-website/pull/13108